dbbackup/management/commands/_base.py

```diff
@@ -0,0 +1,18 @@
+from optparse import make_option
+from django.core.management.base import BaseCommand, LabelCommand
+
+
+class BaseDbBackupCommand(LabelCommand):
```

Comment:
You inherit from LabelCommand, right? That command class behaves in a certain way, namely that it expects you to pass a label of something.

The commands in django-dbbackup do not use labels. If you wanted to add that, it would break the API afaik...?